test/cypress/integration/36-pledges.text.js

```diff
@@ -0,0 +1,85 @@
+describe('Pledges', () => {
+  beforeEach(() => {
+    cy.login({ redirect: `/pledges/new` });
+  });
+
+  describe('creating an individual pledge', () => {
+    it('Create the pledge', () => {
+      cy.get('[data-cy=name]', { timeout: 600000 }).clear().type('samcaspus');
+      cy.get('[data-cy=slug]', { timeout: 600000 }).clear().type('opencollective');
+      cy.get('[data-cy=githubHandle]', { timeout: 600000 }).clear().type('opencollective/opencollective-frontend');
+      cy.get('[id=publicMessage]', { timeout: 600000 }).clear().type('publicMessage');
+      cy.get('[data-cy="submit"]', { timeout: 600000 }).click();
+      cy.get('[data-cy="pledgeStats"]', { timeout: 600000 }).contains('$');
+      cy.get('[data-cy="pledgeStats"]', { timeout: 600000 }).should('contain', '$');
+    });
+
+    it('join an existing pledge individual pledge', () => {
```

Comment:
This test should directly open the page for an existing pledged collective (ie. https://opencollective.com/duckduckgo) and click on `Make a pledge` to join it rather to try and create a new one.

Reply:
made the change